tests/testthat/test-utils.R

```diff
@@ -24,5 +24,21 @@ test_that('validate_log_level', {
     expect_error(logger:::validate_log_level('FOOBAR'), 'log level')
 })
 
+test_that('catch_base_log', {
+    expect_true(nchar(logger:::catch_base_log(ERROR, NA_character_)) == 28)
+    expect_true(nchar(logger:::catch_base_log(INFO, NA_character_)) == 27)
+    layout_original <- log_layout()
+    log_layout(layout_blank)
+    expect_true(nchar(logger:::catch_base_log(INFO, NA_character_)) == 0)
+    log_layout(layout_original)
+    layout_original <- log_layout(namespace = 'TEMP')
+    logger <- layout_glue_generator(format = '{node}/{pid}/{namespace}/{fn} {time} {level}: {msg}')
```

Comment:
I suspect the PR builder checks (tests) are failing due to this .. `nod` and `pid` can have variable with, maybe test without those?

Reply:
see e.g. https://github.com/daroczig/logger/runs/4681051142?check_suite_focus=true

Reply:
I change the computer and now it works on both of them. You are right node and pid have to be removed from tests as are machine specific. Good finding, you save a lot of my time:) Of course now it looks to be obvious.

Reply:
![image](https://user-images.githubusercontent.com/10676545/147862825-f51fa58c-f9d9-4531-9ce8-4d1683029c34.png)

Reply:
ah looks like default `time` has different format on old Linux machine. I will remove it from layout and we should be at home. Rest of machines are ok.